docs/fireflyOverView.md

```diff
@@ -0,0 +1,219 @@
+
+An important feature is that the Firefly Tools server can be installed cross-site. In other words, it is not required to be on the same server as the web page. Firefly Tools can do this because it uses JSONP or CORS for the server communication. This allows Firefly Tools not to be limited by the server's Same Origin Policy and to give the developer a lot of flexibility. The web developer does not have to do any installation, but can simply just start using Firefly Tools.
+
+### Visualizers
+The firefly components contain 3 main visualizers for astronomy data.
+
+
+#### Tabular Display
+Firefly has implemented “Excel-like” tables on the webpage. In an easy, interactive way, a user can sort the results, filter the data on multiple constraints, hide or show columns or select data for download or visualization. The Firefly Tools server is optimized to show very large tables without significant performance degradation. We have tested it with over 10 million rows, 50 column tables.
+
+#### FITS Visualization
+Firefly provides a first-class FITS visualization on the Web without any plugins. All of the significant components you would expect to see in a basic desktop FITS application are available with data that Firefly displays. The FITS visualizer allows for plotting anything that the table shows that has a Lon and Lat. Therefore, users can overlay multiple catalogs over their FITS image and interact with it.
+
+#### 2D Line Graphs
+Firefly shows 2D line graphs interactively so that a user can read the data point values as he moves his mouse around or zooms in to investigate the data at a finer level. These graphs are used for spectrum or plotting table columns. The user can specify any column that the tables are showing. The user can also define equations from the rows for the XY Plot, such as (row1*row2/tan(row3))
+
+## Branches
+There are several branches the this repository.  Here are the ones that you should care about.
+
+ - rc: the next release canidate branch.  This is the most stable branch.
+ - master: Update once a week.  Used for out stable, weekly build.
+ - dev: this branch is could be updated every day. It is the active development branch. It is not stable.
+ - x-??? : Any branch that begins with x- is temporary and used development feature branches
+
+
+## Setup
+
+#### Prerequisites
+ -  [Java 1.8] http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
+    This is the version we are using to compile and run Firefly.
+
+ -  [Gradle 2.3+] (https://gradle.org/downloads)
+    Gradle is an open source build automation system.
+
+
+ -  [Tomcat 7+] (http://tomcat.apache.org/download-70.cgi)
+    Apache Tomcat is an open source software implementation of the Java Servlet and JavaServer Pages technologies.
+
+ -  [Node 0.12.0.+] (https://nodejs.org/download/)
+    Javascript interpreter for command line environment, used for development tools
+
```

Comment:
The 4 items above do not show as links. Perhaps, we should put them in the form `[text](link)` with no spaces. Is this file replacing README.md?